### ISSUE TITLE
Author Metadata: Xinpeng Wang

### ISSUE THREAD

**Issue by Xinpeng-Wang** [acl-org/acl-anthology#3541]
### Author Pages

https://aclanthology.org/people/x/xinpeng-wang/

### Author ORCID

https://orcid.org/0009-0006-5213-1119 

### Institution of highest (anticipated) degree

LMU Munich

### Your papers (if required, see comment below)

https://aclanthology.org/2025.acl-long.90/
https://aclanthology.org/2024.findings-acl.441/
https://aclanthology.org/2024.findings-emnlp.513/
https://aclanthology.org/2024.findings-emnlp.842/
https://aclanthology.org/2023.acl-short.157/
https://aclanthology.org/2023.emnlp-main.126/

### Type of Author Metadata Correction

- [X] The author page wrongly conflates different people with the same name.
- [ ] This author has multiple pages with different spellings of their name.
- [ ] This author has permanently changed their name.

### Supporting Information

I’m the author of https://aclanthology.org/2023.emnlp-main.126/ and https://aclanthology.org/2023.acl-short.157/, but not the others.
Can you help me create a separate link? The current link is already used in open review.
My google scholar: https://scholar.google.com/citations?user=QcNNM2YAAAAJ&hl=en

**Comment by mjpost** [comment]
Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.

If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

**Comment by Xinpeng-Wang** [comment]
> Hello: we are attempting to close out a large backlog of author page requests. As part of these efforts, we are collecting additional information ([ORCID](https://orcid.org/) and degree institution) which will help us assign papers to the correct author in the future. Please modify the updated description above with the requested information.
> 
> If you are requesting to split an author page (i.e., your page has some papers that are not yours), please also provide a list of your papers, in the form of Anthology IDs or URLS (e.g., 2023.wmt-1.13 or https://aclanthology.org/2023.wmt-1.13/). We are unable to match papers to Google or Semantic Scholar profiles.

Hi, I have updated the info above. Thanks!

**Comment by mjpost** [comment]
Can you please review your author page under the preview link posted to the PR? It should be ready in ~30 minutes.

**Comment by Xinpeng-Wang** [comment]
> Can you please review your author page under the preview link posted to the PR? It should be ready in ~30 minutes.

The preview link doesn't show my page. It shows the ACL main page.

**Comment by mjpost** [comment]
Yes, you have to navigate to your author page under the preview: 
- https://preview.aclanthology.org/author-page-xinpeng-wang-lmu/people/xinpeng-wang-lmu/
- https://preview.aclanthology.org/author-page-xinpeng-wang-lmu/people/xinpeng-wang

**Comment by Xinpeng-Wang** [comment]
Thanks, the preview looks good to me.
